app.js

```diff
@@ -50,52 +50,69 @@ var countries = ['IRELAND', 'MEXICO', 'CANADA'];
 var userCountryGuesses = [];
 var score = 0;
 
+//--------------------
+
 //Now the game begins. Using a for loop to access the questions array then returning a response from either the correct or incorrect answer response arrays using an if/else statement. Push user's answer to the userCorrectAnswers array if, well, correct.
-for (var i = 0; i < 5; i++) {
-  userAnswers.push(prompt(questions[i]));
-  if(userAnswers[i].toUpperCase() === answers[i]){
-    console.log(correctAnswerResponses[i]);
-    score++;
-  } else {
-    console.log(incorrectAnswerResponses[i]);
+function questionOneToFive(){
+  for (var i = 0; i < 5; i++) {
+    userAnswers.push(prompt(questions[i]));
+    if(userAnswers[i].toUpperCase() === answers[i]){
+      console.log(correctAnswerResponses[i]);
+      score++;
+    } else {
+      console.log(incorrectAnswerResponses[i]);
+    }
   }
 }
+questionOneToFive();
+//-------------------------------
 
-for(var guesses = 0; guesses < 4; guesses++){
-  var guess = prompt('How old am I?');
-  var guess = parseInt(guess, 10);
-  userNumberGuesses.push(guess);
-  if(userNumberGuesses[guesses] === ranNumber){
-    console.log(correctAnswerResponses[5]);
-    userAnswers.push(userNumberGuesses[guesses]);
-    score++;
-    break;
-  } else if(userNumberGuesses[guesses] > ranNumber){
-    console.log(incorrectSixHigh);
-  } else if(userNumberGuesses[guesses] < ranNumber){
-    console.log(incorrectSixLow);
-  } else {
-    console.log('I don\'t recognize that answer.');
-    userNumberGuesses.splice(-1, 1); // Need to remove last guess, or else it will get stuck in the loop.
-    guesses--; //Need to lower the number, or the game can end with non-number guesses.
-  }
-  if(guesses === 3){
-    console.log(incorrectSix);
+function questionSixAsk(){
+
+  for(var guesses = 0; guesses < 4; guesses++){
+    var guess = prompt('How old am I?');
+    var guess = parseInt(guess, 10);
+    userNumberGuesses.push(guess);
+    if(userNumberGuesses[guesses] === ranNumber){
+      console.log(correctAnswerResponses[5]);
+      userAnswers.push(userNumberGuesses[guesses]);
+      score++;
+      break;
+    } else if(userNumberGuesses[guesses] > ranNumber){
+      console.log(incorrectSixHigh);
+    } else if(userNumberGuesses[guesses] < ranNumber){
+      console.log(incorrectSixLow);
+    } else {
+      console.log('I don\'t recognize that answer.');
+      userNumberGuesses.splice(-1, 1); // Need to remove last guess, or else it will get stuck in the loop.
+      guesses--; //Need to lower the number, or the game can end with non-number guesses.
+    }
+    if(guesses === 3){
+      console.log(incorrectSix);
+    }
   }
 }
+questionSixAsk();
```

Comment:
It'll be a lot easier for you if you group your function calls somewhere so that you can find them.